deploy/kubernetes/elastic-agent-standalone-kubernetes.yaml

```diff
@@ -692,16 +692,18 @@ spec:
       # Uncomment if using hints feature
       #initContainers:
       #  - name: k8s-templates-downloader
-      #    image: busybox:1.28
```

Comment:
reason: with `busybox:1.28` error:
```
wget: TLS error from peer (alert code 80): 80
wget: error getting response: Connection reset by peer
```

with the latest busybox image:
```
Connecting to github.com (140.82.121.3:443)
wget: note: TLS certificate validation not implemented
```
it is not an error, see https://github.com/docker-library/busybox/issues/64 but as suggested in it should be used image with openssl and ca-certificates.
Option would be to use an alpine image, but since `docker.elastic.co/beats/elastic-agent` has curl that can be used to download templates. I think it makes sense to remove extra image dependency